pages/linux/toilet.md

```diff
@@ -0,0 +1,20 @@
+# toilet
+
+> A tool to display ASCII-art fonts.
+> More information: <http://caca.zoy.org/wiki/toilet>.
+
+- Generate ASCII art by directly inputting text:
```

Comment:
"inputting text" is not a word. How about - "Generate ASCII art for a given text" ?